<launch>
  <arg name="username" default="dummyusername" />
  <arg name="password" default="dummypassword" />
  <arg name="app_token" default="/home/administrator/.bosdyn/dev.app_token" />
  <arg name="hostname" default="192.168.131.3" />

  <include file="$(find spot_description)/launch/description.launch" />

  <node pkg="spot_driver" type="spot_ros.py" name="spot_ros" >
    <rosparam file="$(find spot_driver)/config/spot_ros.yaml" command="load" />
    <param name="username" value="$(arg username)" />
    <param name="password" value="$(arg password)" />
    <param name="app_token" value="$(arg app_token)" />
    <param name="hostname" value="$(arg hostname)" />
  </node>

</launch>
